docs/book/src/plugins/go-v2-plugin.md

```diff
@@ -0,0 +1,51 @@
+# go/v2 plugin
+
+The `go/v2` plugin has the purpose to scaffold Golang projects to help users to build projects with [controllers][controller-runtime] and keep the backwards compatibility with the default scaffold made using Kubebuilder CLI `2.x.z` releases.   
```

Comment:
"...to scaffold Golang projects to help users..." -> "... to scaffold Golang projects by helping users..."

Reply:
"...to scaffold..." -> "...of scaffolding..."